src/deprecated.jl

```diff
@@ -1325,8 +1325,23 @@ end
 import Base: vcat
 @deprecate vcat(x::Vector{<:AbstractDataFrame}) vcat(x...)
 
-@deprecate showcols(df::AbstractDataFrame, all::Bool=false, values::Bool=true) describe(df, stats = [:eltype, :nmissing, :first, :last])
-@deprecate showcols(io::IO, df::AbstractDataFrame, all::Bool=false, values::Bool=true) show(io, describe(df, stats = [:eltype, :nmissing, :first, :last]), all)
+@deprecate showcols(df::AbstractDataFrame, all::Bool=false, values::Bool=true) describe(df, :eltype, :nmissing, :first, :last)
+@deprecate showcols(io::IO, df::AbstractDataFrame, all::Bool=false, values::Bool=true) show(io, describe(df, :eltype, :nmissing, :first, :last), all)
+function StatsBase.describe(df::AbstractDataFrame; stats=nothing)
```

Comment:
Doesn't this function overwrite the one defined above, printing a warning? If so, the other one should be commented out for now.

Reply:
It does overwrite, but I thought that the warning was not printed. But I think that commenting it out for now with TODO note is best.

Reply:
Sorry im a bit confused. What do I comment out and which TODO do I write?

Reply:
Comment out the overwritten function and above it write TODO to remember to uncomment it when the deprecation period is finished. Have a look at abstractdataframe/iteration.jl for an example  (nothing is commented out there but this is the idea). When we have such TODO notes it is easy to grep the whole repository to find things that need fixing.

Reply:
by "overwritten function" I mean "overwritten method definition" to be precise.

Reply:
Done. I think I did this right but let me know.

Reply:
Thank you 👍. This is what will allow to avoid the problem @nalimilan mentioned above.